packages/compass-components/package.json

```diff
@@ -31,39 +31,39 @@
     "reformat": "npm run prettier -- --write ."
   },
   "dependencies": {
-    "@leafygreen-ui/badge": "^6.1.2",
-    "@leafygreen-ui/banner": "^5.0.3",
-    "@leafygreen-ui/button": "^16.1.1",
-    "@leafygreen-ui/card": "^7.1.1",
-    "@leafygreen-ui/checkbox": "^10.1.1",
-    "@leafygreen-ui/code": "^12.1.1",
-    "@leafygreen-ui/combobox": "^3.1.1",
-    "@leafygreen-ui/confirmation-modal": "^3.1.1",
+    "@leafygreen-ui/badge": "^8.0.0",
+    "@leafygreen-ui/banner": "^7.0.0",
+    "@leafygreen-ui/button": "^19.0.0",
+    "@leafygreen-ui/card": "^9.0.0",
+    "@leafygreen-ui/checkbox": "^12.0.0",
+    "@leafygreen-ui/code": "^14.0.0",
+    "@leafygreen-ui/combobox": "^5.0.0",
+    "@leafygreen-ui/confirmation-modal": "^3.1.4",
     "@leafygreen-ui/emotion": "^4.0.3",
-    "@leafygreen-ui/icon": "^11.11.1",
-    "@leafygreen-ui/icon-button": "^13.2.1",
-    "@leafygreen-ui/leafygreen-provider": "^2.3.5",
-    "@leafygreen-ui/logo": "^6.3.0",
-    "@leafygreen-ui/marketing-modal": "^3.0.7",
-    "@leafygreen-ui/menu": "^16.0.0",
-    "@leafygreen-ui/modal": "^10.1.2",
+    "@leafygreen-ui/icon": "^11.12.0",
+    "@leafygreen-ui/icon-button": "^15.0.0",
+    "@leafygreen-ui/leafygreen-provider": "^3.1.0",
+    "@leafygreen-ui/logo": "^7.0.0",
+    "@leafygreen-ui/marketing-modal": "^3.0.10",
+    "@leafygreen-ui/menu": "^19.0.0",
+    "@leafygreen-ui/modal": "^12.0.0",
     "@leafygreen-ui/palette": "^3.4.4",
-    "@leafygreen-ui/pipeline": "^3.0.0",
-    "@leafygreen-ui/popover": "^9.1.1",
-    "@leafygreen-ui/portal": "^4.0.6",
-    "@leafygreen-ui/radio-box-group": "^9.1.1",
-    "@leafygreen-ui/radio-group": "^8.1.1",
-    "@leafygreen-ui/segmented-control": "^3.1.1",
-    "@leafygreen-ui/select": "^7.1.2",
-    "@leafygreen-ui/table": "^6.1.1",
-    "@leafygreen-ui/tabs": "^9.2.1",
-    "@leafygreen-ui/text-area": "^6.2.1",
-    "@leafygreen-ui/text-input": "^10.2.1",
-    "@leafygreen-ui/toast": "^2.0.3",
-    "@leafygreen-ui/toggle": "^8.0.5",
+    "@leafygreen-ui/pipeline": "^5.0.0",
+    "@leafygreen-ui/popover": "^11.0.0",
+    "@leafygreen-ui/portal": "^4.0.7",
+    "@leafygreen-ui/radio-box-group": "^12.0.0",
```

Comment:
We can now pass a `darkMode` prop to this component. Let's wrap it like the other components in `leafygreen.tsx`? Looks like we can also wrap `Menu` which we aren't doing.

Reply:
Done in the newest commit, also bumped the versions to latest. Wrapping the `Menu` in this pr: https://github.com/mongodb-js/compass/pull/3851/files#diff-5a823b9048a6c37e36b7e11820ad700524a712cfdf992bae9f3775ad5a2c3c57R167